CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/utils/DataAccessPreferenceUtils.java

```diff
@@ -24,12 +24,17 @@
  *
  * @author cygnus_x-1
  */
-public final class DataAccessPreferenceKeys {
-
+public final class DataAccessPreferenceUtils {
```

Comment:
Can you please end it with "Utilities". We've got a naming convention thing going on specifically when we have utility classed. See point 8 on https://github.com/constellation-app/constellation/blob/master/CHECKLIST.md. Thank you.